docs/05-technical-reference/svls-08-function-specification.md

```diff
@@ -109,9 +109,41 @@ See the detailed descriptions of these fields:
 | **data** | Either JSON or a string, depending on the request type. Read more about [Buffer](https://nodejs.org/api/buffer.html) in Node.js and [bytes literals](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals) in Python. |
 | **extensions** | JSON object that can contain event payload, a Function's incoming request, or an outgoing response |
 
+### Event object SDK
+
+The `event` object is extended by functions making some operations easier. You can use every function by providing `event.{FUNCTION_NAME(ARGUMENTS...)}`.
```

Comment:
```suggestion
The `event` object is extended by Functions making some operations easier. You can use every Function by providing the following: `event.{FUNCTION_NAME(ARGUMENTS...)}`.
```
Is `event` an object or an argument? See the **Signature** section -> 

"Functions must accept two arguments that are passed to the Function handler:
- `event`
- `context`"